packages/api/package.json

```diff
@@ -17,7 +17,7 @@
     "build": "yarn workspace loot-core build:api"
   },
   "dependencies": {
-    "better-sqlite3": "^7.5.0",
+    "better-sqlite3": "^8.1.0",
```

Comment:
Do unit tests still pass? Last time I tried to do this for the electron upgrade some tests started mysteriously failing.

Reply:
Here's my _very thorough_ investigation: https://github.com/actualbudget/actual/pull/462#discussion_r1070625353 😅

Reply:
Will see once your PR is merged and the CI runs as expected

Reply:
I see what you mean. Fun, reverted that change

Reply:
Did some testing and the test is fine with v7.4.6 but breaks with v7.5.0. Nothing jumps out in the diff.

Reply:
Diff: https://github.com/WiseLibs/better-sqlite3/compare/v7.4.6...v7.5.0

My initial guess would be that the upgrade of sqlite to v3.37.2 is at fault here. Not sure how though.. and not sure if even the tests we currently have are correct. They might be incorrect in `master` branch and the upgrade is just exposing the problem.

Reply:
Figured out the underlying issue: With SQLite 3.37.0, the transaction IDs fetched in the second call to `updateCategoryRules` in the test are `[ 'five', 'four', 'six', 'one', 'three', 'two' ]` but with 3.37.2 the IDs are `[ 'one', 'two', 'three', 'four', 'five', 'six' ]`. Reverting that change and will explore in a future PR… turns out the order rows are returned is implementation defined unless you specify `ORDER BY`.

Reply:
Good investigation! I was trying to figure it out too just now, but didn't get as far as you :)